test/var_sets_test.py

```diff
@@ -4,6 +4,7 @@
 
 from .base import TestTFCBaseTestCase
 
+TFC_PAGE_SIZE = 20
```

Comment:
Don't need to redefine this, you can pull the default from the [_constants](https://github.com/dahlke/terrasnek/blob/main/terrasnek/_constants.py) file. Example [here](https://github.com/dahlke/terrasnek/blob/14c84a87b3f8f80ede3ed4b0fb03cc4464a527cf/test/admin_users_test.py#L6).

Reply:
Changed this to import from the the existing file.